src/node_api.cc

```diff
@@ -1184,20 +1195,20 @@ napi_status napi_define_class(napi_env env,
     }
   }
 
-  v8::Local<v8::Value> v8_result_value = tpl->GetFunction();
-  v8::Local<v8::Object> v8_result_object = v8_result_value.As<v8::Object>();
-  *result = v8impl::JsValueFromV8LocalValue(scope.Escape(v8_result_value));
-
-  if (static_property_count > 0) {
+  if (static_property_count > 0) { */
```

Comment:
Same here code, likely needs to be removed or uncommented.

Reply:
I guess never mind since I was just looking at a single commit and I think its not there when all commits are reviewed.